src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+Squid.kt

```diff
@@ -87,3 +87,16 @@ internal fun TradingStateMachine.squidStatus(
     transferStatuses = routerProcessor.receivedStatus(transferStatuses, json, transactionId)
     return StateChanges(iListOf(Changes.transferStatuses))
 }
+
+internal fun TradingStateMachine.routerTrack(
+    payload: String,
+): StateChanges? {
+    val json = try {
+        Json.parseToJsonElement(payload).jsonObject.toMap()
+    } catch (exception: SerializationException) {
+        Logger.e { "Failed to deserialize skipTrack: $payload \nException: $exception" }
+        return StateChanges(iEmptyList())
+    }
```

Comment:
maybe can use ParserProtocol's

`decodeJsonObject` to reduce this bit

Reply:
decodeJsonObject performs an extra conversion to `IMap` which is slower than the native kotlin map. if you want i can make a `decodeJsonObjectToNativeMap` or something